pkg/cmd/cmd.go

```diff
@@ -232,8 +232,18 @@ func _config(config *pomo.Config) func(*cli.Cmd) {
 	}
 }
 
+//cli description
+func appDescription() (name string, description string) {
```

Comment:
In order to test it I moved command declaration to separate method , I think about doing it for all commands, @kevinschoon  what do you think ?

Reply:
I took a closer look at your code and I see you want to run the CLI tests with a "blackbox" style, I hadn't thought of doing that before but seems like it could work nicely.

Maybe we could change the signature of the `Run` like below, and then each test can just pass arguments like this:

```go
func Run(config *pomo.Config, args... []string) error {
        app := cli.App(appDescription())
        ....
```

Then the structure of each test would look like this:

```go
func TestDeleteTasks(t *testing.t) {
    err = Run(config "pomo", "delete", "1")
    if err != nil {
        t.Error(err)
    }
    // check database here
   ....
}
```
That would require us to restructure the code a little bit but it's probably the only way to do the tests correctly.

As I mentioned before if you'd like we can just verify the delete functionality locally and restructure the code for tests in another PR, or if you're up to it then go ahead and refactor the code in this PR for your delete test.

Thank you @strogiyotec!